runners/gradle-plugin/build.gradle.kts

```diff
@@ -8,15 +8,14 @@ plugins {
 }
 
 dependencies {
-    api(project(":core"))
-
-    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin")
-    compileOnly("com.android.tools.build:gradle:4.0.1")
-    compileOnly(gradleKotlinDsl())
```

Comment:
I removed `compileOnly(gradleKotlinDsl())` because it's added by the `kotlin-dsl` plugin